thirdeye-ui/src/app/components/entity-cards/anomaly-card/anomaly-card.component.tsx

```diff
@@ -40,7 +40,7 @@ export const AnomalyCard: FunctionComponent<AnomalyCardProps> = (
         <Card className={props.className} variant="outlined">
             <CardContent>
                 {props.uiAnomaly && (
-                    <Grid container spacing={4}>
+                    <Grid container justifyContent="space-between" spacing={4}>
```

Comment:
Does spece-between work if there are xs, sm, and lg defined?

Reply:
Yep. It helps in evenly distributing space between the cells, which is more prominent on larger displays. Without this property, all the cells are stacked to the left and the empty space is disproportionately stacked to the right.


On large screens, having the property looks like this (on local)
![Screenshot_36](https://user-images.githubusercontent.com/20799363/212001857-7c3a57ba-7409-4aeb-9b9b-cf22bc27aac3.png)

Whereas, without having it (on teprod)
![Screenshot_37](https://user-images.githubusercontent.com/20799363/212001932-89e12cc4-6f8b-49c1-ab71-e190eb2870f1.png)